code/modules/mob/living/simple_animal/vore/rous.dm

```diff
@@ -43,4 +43,117 @@
 
 /mob/living/simple_animal/hostile/rous/death()
 	playsound(src, 'sound/effects/mouse_squeak_loud.ogg', 50, 1)
+	if(resting)
+		resting = 0
+		icon_state = icon_dead
+		update_icon()
 	..()
+
+/mob/living/simple_animal/hostile/rous/passive
+	name = "curious giant rat"
+	desc = "In what passes for a hierarchy among verminous rodents, this one is king. It seems to be more interested on scavenging."
+	follow_dist = 1
+	var/mob/living/carbon/human/food
+	var/hanger = 0
```

Comment:
Please use variables and pathing names that make sense and are spelled correctly. `hanger` should be `hunger` for ease of reading by other devs. Memey code is not acceptable.